setup.py

```diff
@@ -491,6 +491,7 @@ def fixed_readme() -> str:
             "pip-tools<6.11",
             "pybind11-stubgen",
             "pylint",
+            "lazy-object-proxy>=1.9.0",
```

Comment:
Should note that this is not a direct dependency, is a transitive dependency of pylint, and is only listed here to pin the version

Reply:
Done.